x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/calculations/differences.tsx

```diff
@@ -135,5 +135,13 @@ Example: Visualize the change in bytes received over time:
       `,
     }),
   },
+  quickFunctionDocumentation: i18n.translate(
+    'xpack.lens.indexPattern.differences.documentation.quick',
+    {
+      defaultMessage: `
+      The change, or difference, between the values in subsequent intervals.
```

Comment:
```suggestion
      The change between the values in subsequent intervals.
```
Wouldn't it be better this way? I find the `or difference` more confusing than helping